src/components/Rankings.js

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { getAllFieldCoveredCounts } from '../lib/coverageStats/statsUtils';
+
+const SectionColors = {
+  Patient: 'fill-[#D24200]',
+  Outcome: 'fill-[#8A45D9]',
+  Disease: 'fill-[#F2B84B]',
+  Treatment: 'fill-[#04B2D9]',
+  Assessment: 'fill-[#F2913D]',
+  Genomics: 'fill-[#26C485]',
+};
```

Comment:
Let's use the tailwind variables that Alden defined in our `tailwind.config.js` file. The colors defined in the theme.extend.colors object can be used as you would any other color; i.e. `fill-patient` would correspond to `fill-[#d24200`